nrf_802154/sl/sl_opensource/src/nrf_802154_sl_fem.c

```diff
@@ -121,6 +122,11 @@ bool mpsl_fem_prepare_powerdown(NRF_TIMER_Type * p_instance,
     return false;
 }
 
+bool mpsl_fem_device_config_254_shall_apply_get(void)
```

Comment:
Why is it redefined here?

Reply:
It is a stub definition for open source SL version which does not use the MPSL.